pkg/controlplane/control/manager.go

```diff
@@ -123,6 +192,32 @@ func (m *Manager) AddImport(ctx context.Context, imp *v1alpha1.Import) error {
 	return err
 }
 
+// deleteClDataplane deletes pod to ipToPod list.
```

Comment:
See previous comment regarding watching the EndpointSlice of the cl-dataplane service instead of working at the level of individual Pods.